frontend/src/scenes/apps/HistoricalExportsTab.tsx

```diff
@@ -1,65 +1,108 @@
-import { useValues } from 'kea'
+import { useActions, useValues } from 'kea'
 import { appMetricsSceneLogic, HistoricalExportInfo } from './appMetricsSceneLogic'
 import { LemonTable, LemonTableColumn } from 'lib/components/LemonTable'
 import { HistoricalExport } from './HistoricalExport'
 import { createdAtColumn, createdByColumn } from 'lib/components/LemonTable/columnUtils'
 import { LemonTag } from 'lib/components/LemonTag/LemonTag'
 import { Progress } from 'antd'
+import { LemonButton } from 'lib/components/LemonButton'
+import { PluginJobModal } from 'scenes/plugins/edit/interface-jobs/PluginJobConfiguration'
+import { useEffect } from 'react'
+
+const RELOAD_HISTORICAL_EXPORTS_FREQUENCY_MS = 20000
 
 export function HistoricalExportsTab(): JSX.Element {
-    const { historicalExports, historicalExportsLoading, pluginConfig } = useValues(appMetricsSceneLogic)
+    const { historicalExports, historicalExportsLoading, pluginConfig, interfaceJobsProps, hasRunningExports } =
+        useValues(appMetricsSceneLogic)
+    const { openHistoricalExportModal, loadHistoricalExports } = useActions(appMetricsSceneLogic)
+
+    useEffect(() => {
+        let timer: NodeJS.Timeout | undefined
+
+        function updateTimer(): void {
+            if (hasRunningExports) {
+                timer = setTimeout(() => {
+                    loadHistoricalExports()
+                    updateTimer()
+                }, RELOAD_HISTORICAL_EXPORTS_FREQUENCY_MS)
+            }
+        }
+
+        updateTimer()
+        return () => timer && clearTimeout(timer)
+    }, [hasRunningExports])
 
     return (
-        <LemonTable
-            dataSource={historicalExports}
-            loading={historicalExportsLoading}
-            columns={[
-                {
-                    title: 'Dates exported',
-                    render: function Render(_, historicalExport: HistoricalExportInfo) {
-                        const [dateFrom, dateTo] = historicalExport.payload.dateRange
-                        if (dateFrom === dateTo) {
-                            return dateFrom
-                        }
-                        return `${dateFrom} - ${dateTo}`
+        <div className="space-y-2">
+            <div className="flex items-center justify-end">
+                <LemonButton type="primary" onClick={openHistoricalExportModal} disabled={!interfaceJobsProps}>
+                    Start new export
+                </LemonButton>
+            </div>
+
+            <LemonTable
+                dataSource={historicalExports}
+                loading={historicalExportsLoading}
+                columns={[
+                    {
+                        title: 'Dates exported',
+                        render: function Render(_, historicalExport: HistoricalExportInfo) {
+                            const [dateFrom, dateTo] = historicalExport.payload.dateRange
+                            if (dateFrom === dateTo) {
+                                return dateFrom
```

Comment:
add a tooltip here with full dates with time zones

Reply:
Unrelated to this change - this was a whitespace change.